bridge_adaptivity/bridge_lti/provider.py

```diff
@@ -110,20 +118,19 @@ def create_sequence_item(request, sequence, start_activity, tool_provider, lti_c
     sequence_item = SequenceItem.objects.create(
         sequence=sequence,
         activity=start_activity,
-        position=1
+        position=1,
     )
     return sequence_item
 
 
-def learner_flow(request, lti_consumer, tool_provider, collection_slug=None, group_slug=None, unique_marker=''):
+def learner_flow(request, lti_consumer, tool_provider, collection_order_slug=None, unique_marker=''):
     """
     Define logic flow for Learner.
     """
-    if not collection_slug:
+    if not collection_order_slug:
         return stub_page(request)
 
-    collection, collection_group, engine = get_collection_collectiongroup_engine(collection_slug, group_slug)
-
+    engine, collection_order = get_collection_collection_order_engine(collection_order_slug)
```

Comment:
Please rename function removing inappropriate parts.

Reply:
I fixed it.